config/store.go

```diff
@@ -4,9 +4,10 @@ import "github.com/qri-io/jsonschema"
 
 // Store configures a qri content addessed file store (cafs)
 type Store struct {
-	Type    string                 `json:"type"`
-	Options map[string]interface{} `json:"options,omitempty"`
-	Path    string                 `json:"path,omitempty"`
+	Type        string                 `json:"type"`
+	Options     map[string]interface{} `json:"options,omitempty"`
+	Path        string                 `json:"path,omitempty"`
+	Filesystems []string               `json:"Filesystems,omitempty"`
```

Comment:
Is this intended as a stepping stone for configuration? The RFC specs out `Filesystems` as a top-level config field structured like this:

```go
type Filesystem struct {
  Type string
  // Source string // TODO - enable once pluggable sources are a thing
  Config map[string]interface{}
}
```

```go
type Config struct {
  // ...
  Filesystems []Filesystem
  // ...
}
```

Reply:
Let's pretend I didn't zone out completely while doing this...

Reply:
should we pull in the `MuxConfig` struct for this?

Note also @Arqu we renamed `Options` to `Config` from the rfc

```
// MuxConfig contains the information needed to create a new filesystem
type MuxConfig struct {
	Type   string                 `json:"type"`
	Config map[string]interface{} `json:"config,omitempty"`
	Source string                 `json:"source,omitempty"`
}
```

https://github.com/qri-io/qfs/blob/8af27780e44bbeefa98491c7146450a256b246b9/muxfs/mux.go#L47-L52